.github/workflows/windows.yml

```diff
@@ -340,6 +340,12 @@ jobs:
           name: ${{ needs.openvino_download.outputs.ov_artifact_name }}
           path: ${{ env.OV_INSTALL_DIR }}
           merge-multiple: true
+          
+      - name: Setup Python ${{ env.PYTHON_VERSION }}
```

Comment:
why do we need python here?
I suppose it was optional in both GenAI and Tokenizers.

Reply:
Because the ICU data is generated by python tool ( https://github.com/unicode-org/icu/blob/main/icu4c/source/data/makedata.mak#L305